socorro/unittest/external/es/base.py

```diff
@@ -123,14 +117,11 @@
         'is_returned': True,
         'is_mandatory': False,
         'storage_mapping': {
-            'type': 'multi_field',
+            'type': 'string',
             'fields': {
-                'os_name': {
-                    'type': 'string'
-                },
```

Comment:
Can you talk about this change? It looks like we're switching it from a `multi_field` to a `string`, but I'm not sure what this went from and to.